mutalyzer/variant.py

```diff
@@ -0,0 +1,270 @@
+        :arg inserted: Inserted part.
+        :type inserted: object
+        :arg shift: Amount of freedom.
+        :type shift: int
+        """
+        # TODO: Will this container be used for all variants, or only genomic?
+        #       start_offset and end_offset may be never used.
+        self.start = start
+        self.start_offset = start_offset
+        self.end = end
+        self.end_offset = end_offset
+        self.sample_start = sample_start
+        self.sample_start_offset = sample_start_offset
+        self.sample_end = sample_end
+        self.sample_end_offset = sample_end_offset
+        self.type = type
+        self.deleted = deleted
+        self.inserted = inserted
+        self.weight_position = weight_position
+        self.shift = shift
+    #__init__
+
+    def __unicode__(self):
+        """
+        Give the HGVS description of the raw variant stored in this class.
+
+        :returns: The HGVS description of the raw variant stored in this class.
+        :rtype: unicode
+        """
+        if self.type == "unknown":
+            return "?"
+        if self.type == "none":
+            return "="
+
+        description = "{}".format(self.start)
+
+        if self.start != self.end:
+            description += "_{}".format(self.end)
+
+        if self.type != "subst":
+            description += "{}".format(self.type)
+
+            if self.type in ("ins", "delins"):
+                return description + "{}".format(self.inserted)
+            return description
+        #if
+
+        return description + "{}>{}".format(self.deleted, self.inserted)
+    #__unicode__
+
+    def weight(self):
+        if self.type == "unknown":
+            return -1
+        if self.type == "none":
+            return 0
+
+        weight = self.weight_position
+        if self.start != self.end:
+            weight += self.weight_position + extractor.WEIGHT_SEPARATOR
+
+        return weight + weights[self.type] + self.inserted.weight()
+    #weight
+#DNAVar
+
+class ProteinVar(models.ProteinVar):
+    """
+    Container for a protein variant.
+    """
+    def __init__(self, start=0, end=0, sample_start=0, sample_end=0,
+            type="none", deleted=ISeqList([ISeq()]),
+            inserted=ISeqList([ISeq()]), shift=0, term=0):
+        """
+        Initialise the class with the appropriate values.
+
+        :arg start: Start position.
+        :type start: int
+        :arg end: End position.
+        :type end: int
+        :arg sample_start: Start position.
+        :type sample_start: int
+        :arg sample_end: End position.
+        :type sample_end: int
+        :arg type: Variant type.
+        :type type: unicode
+        :arg deleted: Deleted part of the reference sequence.
+        :type deleted: unicode
+        :arg inserted: Inserted part.
+        :type inserted: object
+        :arg shift: Amount of freedom.
+        :type shift: int
+        :arg term:
+        :type term:
```

Comment:
It's not necessary to enumerate all arguments here (the method signature is also shown in the API docs), so better remove the last two lines or add actual information.